package.json

```diff
@@ -92,6 +92,7 @@
     "case-sensitive-paths-webpack-plugin": "2.2.0",
     "copyfiles": "2.1.0",
     "coveralls": "3.0.3",
+    "css-loader": "2.1.0",
```

Comment:
Is this really needed for the antd css require in `webpack.config.json`?

Reply:
Yes. 'module not found' errors where thrown before.